tests/storage/assert_storage.rs

```diff
@@ -50,8 +52,12 @@ impl AssertionStorage {
     }
 
     pub fn update_with_key(&mut self, cluster: &mut Cluster<ServerCluster>, key: &str) {
-        cluster.must_get(key.as_bytes());
-        let region = cluster.get_region(key.as_bytes());
+        self.update_with_key_byte(cluster, key.as_bytes());
+    }
+
+    pub fn update_with_key_byte(&mut self, cluster: &mut Cluster<ServerCluster>, key: &[u8]) {
+        cluster.must_get(key);
```

Comment:
`cluster.query_leader()` is private, but it could be modified to be public. Or a new interface like `must_have_leader()` could be added for this specific purpose.